wiki/markdown/installation.md

```diff
@@ -10,4 +10,113 @@ Binary is released via [jitPack](https://jitpack.io/#eiffel-community/eiffel-rem
 
 The latest REMReM Publish CLI binary can be downloaded via [publish-cli.jar](https://jitpack.io/#eiffel-community/eiffel-remrem-publish).
 
-The latest REMReM Publish Service binary can be downloaded via [publish-service.war](https://jitpack.io/#eiffel-community/eiffel-remrem-publish).
\ No newline at end of file
+The latest REMReM Publish Service binary can be downloaded via [publish-service.war](https://jitpack.io/#eiffel-community/eiffel-remrem-publish).
+
+The latest REMReM Publish uses sepia protocol. To make it compatible prior to sepia protocol versions
+create a file at desired location : path/to/routing-key-overrides.properties
+with the below mapping and pass the file location to the jar as follows
+
+Ex: java -DsemanticsRoutingkeyTypeOverrideFilepath=path/to/routing-key-overrides.properties -jar publish-service.war
```

Comment:
Should be -Dcom.ericsson.eiffel.remrem.publish.messasgebus.semanticsRoutingkeyTypeOverrideFilepath

Reply:
If system property is not set it will take semanticsRoutingkeyTypeOverrideFilepath as in the application properties

Reply:
Yes, works fine.